frontend/styles/colors.ts

```diff
@@ -1,22 +1,59 @@
+import { mapValues } from 'lodash';
+
+import { hexToRgba, hexToPinColor } from '../helpers/colors';
+
+const BLACK = '#00000';
+const WHITE = '#ffffff';
+
 export const COLOR = {
-	PRIMARY_MAIN: '#0C2F54',
+	PRIMARY_MAIN: '#0c2f54',
 	PRIMARY_LIGHT: '#3c5876',
 	PRIMARY_DARK: '#08203a',
 	SECONDARY_MAIN: '#06563c',
 	SECONDARY_LIGHT: '#377763',
 	SECONDARY_DARK: '#043827',
-	WHITE: '#fff',
-	WHITE_87: 'rgba(255, 255, 255, 0.87)',
-	BLACK: '#000',
-	BLACK_6: 'rgba(0, 0, 0, .06)',
-	BLACK_20: 'rgba(0, 0, 0, .2)',
-	BLACK_30: 'rgba(0, 0, 0, .3)',
-	BLACK_40: 'rgba(0, 0, 0, .4)',
-	BLACK_50: 'rgba(0, 0, 0, .5)',
-	BLACK_60: 'rgba(0, 0, 0, .6)',
-	BLACK_80: 'rgba(0, 0, 0, .8)',
+
+	WHITE,
+	WHITE_87: hexToRgba(WHITE, 0.87),
+
+	BLACK,
+	BLACK_6: hexToRgba(BLACK, .06),
+	BLACK_20: hexToRgba(BLACK, .2),
+	BLACK_30: hexToRgba(BLACK, .3),
+	BLACK_40: hexToRgba(BLACK, .4),
+	BLACK_50: hexToRgba(BLACK, .5),
+	BLACK_60: hexToRgba(BLACK, .6),
+	BLACK_80: hexToRgba(BLACK, .8),
+	BLACK_87: hexToRgba(BLACK, .87),
+
 	GRAY: '#f0f0f0',
-	WARNING_LIGHT: '#FFEFEF',
-	WARNING: '#FFD5D5',
+	WARNING_LIGHT: '#ffefef',
+	WARNING: '#ffd5d5',
+	YELLOW: '#ffff36',
+	MAROON: '#800000',
+	CRIMSON: '#dc143c',
+	DARK_ORANGE: '#ff8c00',
+	ORANGE: '#ffa500',
+	GREEN: '#008000',
+	MED_SEA_GREEN: '#3cb371',
+	LIME_GREEN: '#32cd32',
+	LIGHT_GREEN: '#90ee90',
 	RED: '#FF0000'
 };
+
+export const PIN_COLORS = mapValues({
```

Comment:
I think it doesn't make sense, because - in this case - COLORS constant was used to build PIN_COLORS. PIN_COLORS is related to COLORS but contains a different data type. eg.
```COLORS: '#ff0000',
PIN_COLORS: [255, 0, 0] ```
But you can still use it like COLORS eg. `PIN_COLORS.YELLOW`.

Let me know if it's not understandable to you. ;)